histogrammar/resources.py

```diff
@@ -21,20 +21,20 @@
 # Resources lookup file for histogrammar
 
 import pathlib
-from pkg_resources import resource_filename
-import histogrammar as hg
+
+ROOT_DIRECTORY = pathlib.Path(__file__).resolve().parent
```

Comment:
why not upgrade to `importlib_resources` instead?

Using relative paths assumes that the package is extracted into filesystem, right? It may not necessarily be the case sometimes (e.g. wheel, zip, egg?)

Reply:
I'm not familiar with it, will check it out.

Reply:
I switched to using importlib.